apptests/appscenarios/traefik_test.go

```diff
@@ -231,6 +231,14 @@ func assertTraefikEndpoints(t *traefik, podList *corev1.PodList) {
 		return podList.Items, err
 	}).WithPolling(5 * time.Second).WithTimeout(time.Minute).Should(HaveLen(1))
 
+	By("triggering metrics generation on port 8443")
+	res := restClientV1Pods.Get().Resource("pods").
+		Namespace(podList.Items[0].Namespace).
+		Name(podList.Items[0].Name + ":8443").
+		SubResource("proxy").
+		Do(ctx)
+	Expect(res.Error()).To(BeNil())
```

Comment:
```suggestion
	_ = restClientV1Pods.Get().Resource("pods").
		Namespace(podList.Items[0].Namespace).
		Name(podList.Items[0].Name + ":8443").
		SubResource("proxy").
		Do(ctx)
```

This endpoint will return 404, so no need to make an assertion here. The goal is only to trigger metrics generation in traefik.

Reply:
ok will do as suggested